Makefile

```
@@ -1,12 +1,11 @@
 build:
 	pipenv install --dev
 
-lint:
-	pipenv run flake8 . ./sample_loader/tests
-	pipenv check
+docker:
+	docker build -t eu.gcr.io/ssdc-rm-ci/rm/ssdc-rm-toolbox .
 
-test: lint
-	pipenv run pytest --cov-report term-missing --cov . --capture no
+flake:
+	pipenv run flake8
 
-docker: test
-	docker build -t eu.gcr.io/ssdc-rm-ci/rm/ssdc-rm-toolbox .
+check:
+	PIPENV_PYUP_API_KEY="" pipenv check -i 39611 -i 39608 -i 40014
```

```suggestion
	pipenv check
```
No need for any of these ignores